daemon/k8s_watcher.go

```diff
@@ -1969,6 +1967,43 @@ func (d *Daemon) deleteK8sPodV1(pod *v1.Pod) {
 	}
 }
 
+// missingK8sPodV1 returns all pods considered missing, if the IP doesn't exist in
+// the cache or if there is an endpoint associated with that pod that doesn't
+// contain all labels from the pod.
+func (d *Daemon) missingK8sPodV1(m versioned.Map) versioned.Map {
+	missing := versioned.NewMap()
+	for k, v := range m {
+		pod := v.Data.(*v1.Pod)
+		_, exists := ipcache.IPIdentityCache.LookupByIP(pod.Status.PodIP)
```

Comment:
Does this mean we consider all pods missing until the kvstore is connected? What kind of side effects does that have?

Reply:
@tgraf why kvstore? we also use pod watch events to update ipcache. https://github.com/cilium/cilium/pull/5655/files#diff-7c38bf553d4d2690add68b2464b58ef0L1901

Reply:
Please add a comment to indicate this. It's not entirely obvious without understanding some implementation details.

Reply:
I'll add the comment once tests finish to avoid running them again.